components/help-dropdown/help-dropdown.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Service from '../../js/service';
+
+export default class HelpDropdown extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+       helpOptions: []
+     };
+  }
+  componentDidMount() {
+    Service.helpTypes.get().then(options => {
+      this.setState({ 
+        helpOptions: options.map(option => option.name) 
+      });
+    });
+  }
+
+  render() {
+    let renderedHelpTypes = this.state.helpOptions.map(type => {
+      return <option key={type} value={type}>{type}</option>;
+    });
+
+    return (
+      <select className="body-large dropdown" value={ this.props.value } onChange={ (event) => {this.props.helpType(event);} }>
```

Comment:
Let's rename `dropdown` to `help-dropdown` so it matches the component name. If in the future we decide to reuse the styling for other dropdowns on the site we can also make code adjustment accordingly.